packages/sku/lib/packageManager.js

```diff
@@ -16,6 +16,24 @@ const skuArgs = require('../config/args');
 
 /** @typedef {'yarn' | 'pnpm' | 'npm'} SupportedPackageManager */
 
+const supportedPackageManagers = ['yarn', 'pnpm', 'npm'];
+
+/**
+ * @param {string} packageManager
+ * @returns {SupportedPackageManager}
+ */
+const validatePackageManager = (packageManager) => {
```

Comment:
This is technically a fix because we didn't have any validation before, but IMO not worth mentioning in the patch notes.